src/app.js

```diff
@@ -1,17 +1,5 @@
 const quotes = require("./quotes");
 const { getRandomElement } = require("./utils");
-
-const express = require("express");
-const app = express();
-const port = process.env.PORT || 3000;
-
-app.get("/", (req, res) => {
-  res.json(quotes);
-});
-
-app.get("/random", (req, res) => {
-  const quote = getRandomElement(quotes);
-  res.json(quote);
-});
+const router = require = ("./router")
```

Comment:
After importing the router, you need to tell the app to use it

Reply:
thanks, working on it at the moment

Reply:
Are you still planning to work on this part? Once that's fixed I can merge this in.

Reply:
Hiya, yes. I already pushed the updated code.

Reply:
If you run the latest version locally and go to localhost:3000 or localhost:3000/random, you will see the routes are currently not available. 

You need to add `app.use("/", router);` to app.js - that will tell the express app to use the routes defined in the router for requests starting with "/". Since that covers all requests, you could also simplify it to `app.use(router);`.

This is shown at the very end in the express documentation:
https://expressjs.com/en/guide/routing.html#express-router

Please add this line and then I can merge your pull request.

Reply:
Done and pushed to git. I've checked the localhost:3000 and .../random and it showed the quotes so I assume it works correctly now. 

Thanks for your help!

Reply:
Awesome, thanks - I'll go ahead and merge this in now.

Reply:
Thanks! Big day for me, my first PR merged with master!